README.md

```diff
@@ -1 +1,282 @@
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+### Deployment
+
+You can deploy this project using:
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/Unleashedicon)
+- Twitter: [@twitterhandle](https://twitter.com/KipkuruiKelvin3e)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/kelvin-kipkurui-7b50b8252/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **Responsive Design: As more and more users are accessing the web on mobile devices, it's important to ensure that your website looks and works well on all screen sizes. This feature would allow the website to adapt to the screen size of the device it's being viewed on.**
+- [ ] **Accessibility: This feature would make the website easy to navigate and use for users with disabilities. This can include things like providing alternative text for images, using semantic HTML tags, and ensuring that the website's contrast ratio is high enough to be easily read by users with low vision.**
+- [ ] **CSS preprocessors: CSS preprocessors such as Sass or Less will allow you to use variables, nested selectors, mixins and many other features that can make your CSS code more maintainable and organized. This can help you to write cleaner and more modular CSS code, making it easier to update and maintain.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+
+We are excited to present our simple HTML and CSS project, which is a great tool for learning the basics of web development. Your support will help us to continue to develop and improve this project, making it an even more valuable resource for those just getting started with web development
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank microverse community for this project it has been really helpful and i cannot wait for my next project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
```

Comment:
Kindly remove this line because it is just an instruction of what you can put in this section of the readme file